crmsh/utils.py

```diff
@@ -3350,4 +3350,47 @@ def fetch_cluster_node_list_from_node(init_node):
         else:
             cluster_nodes_list.append(tokens[1])
     return cluster_nodes_list
+
+
+def has_sudo_access():
+    """
+    Check if current user has sudo access
+    """
+    rc, _, _ = get_stdout_stderr("sudo -S -k -n id -u")
+    return rc == 0
+
+
+def in_haclient():
+    """
+    Check if current user is in haclient group
+    """
+    return 90 in os.getgroups()
```

Comment:
Magic number should not be used here. We cannot assume haclient's group id is 90.

Reply:
Fixed in #1116